src/components/ListPagination.tsx

```diff
@@ -1,88 +1,39 @@
-import { Box, BoxProps, MenuItem, Pagination, PaginationItem, Select, Typography } from '@mui/material';
-import { useTranslation } from 'react-i18next';
-import { ROWS_PER_PAGE_OPTIONS } from '../utils/constants';
-import { dataTestId } from '../utils/dataTestIds';
-import { ListPaginationCounter } from './ListPaginationCounter';
+import { ReactNode } from 'react';
+import { ListPaginationTop } from './ListPaginationTop';
+import { ListPaginationBottom } from './ListPaginationBottom';
 
-interface ListPaginationProps extends Pick<BoxProps, 'sx'> {
+interface ListPaginationProps {
+  children: ReactNode;
   count: number;
   rowsPerPage: number;
   page: number;
   onPageChange: (newPage: number) => void;
   onRowsPerPageChange: (newRowsPerPage: number) => void;
   rowsPerPageOptions?: number[];
   maxHits?: number; // Default limit of 10_000 hits in ElasticSearch
+  showPaginationTop?: boolean;
 }
 
 export const ListPagination = ({
+  children,
   count,
   rowsPerPage,
   page,
   onPageChange,
   onRowsPerPageChange,
-  rowsPerPageOptions = ROWS_PER_PAGE_OPTIONS,
-  maxHits,
-  sx = {},
+  showPaginationTop,
 }: ListPaginationProps) => {
-  const { t } = useTranslation();
-
-  const maxPages = maxHits ? Math.ceil(maxHits / rowsPerPage) : Infinity;
-  const totalPages = Math.ceil(count / rowsPerPage);
-  const pages = Math.min(maxPages, totalPages);
-
-  const itemsStart = count > 0 ? (page - 1) * rowsPerPage + 1 : 0;
-  const itemsEnd = Math.min(page * rowsPerPage, count);
-
   return (
-    <Box
-      sx={{
-        mx: { xs: '0.25rem', md: '0' },
-        display: 'grid',
-        gridTemplateColumns: { xs: '1fr', sm: 'auto 1fr auto' },
-        gap: '0.25rem 0.5rem',
-        alignItems: 'center',
-        justifyItems: 'center',
-        ...sx,
-      }}
-      data-testid={dataTestId.common.pagination}>
-      <ListPaginationCounter start={itemsStart} end={itemsEnd} total={count} />
-
-      <Pagination
-        sx={{
-          '.MuiPagination-ul': {
-            justifyContent: 'center',
-          },
-        }}
+    <>
+      {showPaginationTop && <ListPaginationTop count={count} rowsPerPage={rowsPerPage} page={page} />}
```

Comment:
denne burde kanskje hatt en liten margin-right, for å unngå å komme inni skjermkanten? `sx={{ mr: '0.5rem' }}` feks

![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/d7ef9df8-a3b5-40f9-bbcd-6a2258bd3d50)